packages/ui/src/components/common/ErrorMessage/index.tsx

```diff
@@ -0,0 +1,14 @@
+import cn from "@ui/src/utils/cn";
+
+interface ErrorMessageProps {
+  message: string | undefined;
```

Comment:
```suggestion
  message?: string;
```
여기 undefined는 사용하지 않아도 될거같아!

Reply:
지금 써준거 급하지 않으면 추후에 리팩토링 브랜치 파서 진행할게~

Reply:
수정완~